client/knode/genesis/generator.go

```diff
@@ -168,13 +184,6 @@ func getNetwork(network string) *big.Int {
 	return chainId
 }
 
-func (gen *generator) addBatchOfPrefundedAccountsIntoGenesis() {
```

Comment:
As I mentioned earlier, we need to know the functional reason of why this was like this in the first place.

Reply:
This is what I get from Puppeth @rgeraldes 

// Add a batch of precompile balances to avoid them getting deleted
	for i := int64(0); i < 256; i++ {
		genesis.Alloc[common.BigToAddress(big.NewInt(i))] = core.GenesisAccount{Balance: big.NewInt(1)}
	}

Do you know what it means?